lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -18,7 +18,7 @@
 <?php if ($is_current): ?>
   <?php echo $copy_vars['owners_rb_subtitle'] ?>
   <?php echo $copy_vars['owners_rb_scholarship'] ?>
-  <?php echo $reportback->caption ?>
+  <?php echo check_plain($reportback->caption) ?>
```

Comment:
I'm not sure what reportback-permalink.tpl.php is for. @aaronschachter is this for your app? How can I see this page?